contributing/issues_list.md

```diff
@@ -0,0 +1,32 @@
+(issues_list)=
+# Issues to Work On
+
+## Repositories
+
+The PyMC project has a lot of different repositories for different uses:
+- PyMC videos (add descriptions & timestamps): https://github.com/pymc-devs/video-timestamps
+- Data Umbrella/PyMC working sessions website: https://github.com/pymc-devs/pymc-data-umbrella
+- PyMC branding: https://github.com/pymc-devs/brand/
+- PyMC code: https://github.com/pymc-devs/pymc/
+- PyMC examples: https://github.com/pymc-devs/pymc-examples
+- PyMC documentation: https://github.com/pymc-devs/pymc/tree/main/docs/source
+- PyMC blog: https://github.com/pymc-devs/pymc.io/issues
+- More... see the full list here: https://github.com/pymc-devs
+
+## List of Issues
+
+We have a [**List of Issues**](https://hackmd.io/bptOCow6Ql6o2K9Brq5lkw?both) that new and returning contributors can work on.
+
+Note: Many issues have multiple parts, so the list is longer than it initially appears.
+
+Please comment on the issue if you wish to work on an specific issue.
+
+Reminder, please include this information in your PRs:
+- `#DataUmbrellaPyMCsprint`
+- @ mention your pair programming partner
+
+## Environment Set-up
+
+1. You can set up your [working environment locally](https://pymc-data-umbrella.xyz/en/latest/sprint/tutorials/environment_setup.html) (on your computer).
```

Comment:
This needs to be updated to use sphinx cross references. Otherwise, people reading a translation of this page will get directed to the english version of the env setup instructions even though there are translations available. Can you send a PR or open an issue?